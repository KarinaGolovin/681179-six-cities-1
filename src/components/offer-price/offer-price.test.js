import React from 'react';
import renderer from 'react-test-renderer';
import {OfferPrice} from './offer-price.jsx';

it(`OfferPrice correctly renders`, () => {
  const component = renderer.create(
      <OfferPrice
        price={1}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});
