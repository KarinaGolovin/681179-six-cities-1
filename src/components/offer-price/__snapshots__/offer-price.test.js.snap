// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferPrice correctly renders 1`] = `
<div
  className="property__price"
>
  <b
    className="property__price-value"
  >
    €
    1
  </b>
  <span
    className="property__price-text"
  >
     night
  </span>
</div>
`;
